### DISCUSSION TITLE
Change font in template

### DISCUSSION THREAD

**Discussion by Dre-Tas** [platframe/platframe#17] [Q&A]
I'm trying to change the font in yuor template, but I can't find where to do it. I see your template uses Sans Pro as default and I'm trying to change to a font I have.

In `src/fonts/Source` I created a folder with the name of the template (CircularStd) where I've put all my .ttf font files. I also deleted the original folder containing the Sans Pro font.
In src\styles\_\modules\text\fonts I created a file called source-circularstd.styl (similar to source-sans-pro.styl) where I'm defining the font files
![image](https://user-images.githubusercontent.com/36023527/70691591-8fa5a780-1d0d-11ea-8d76-bd3f873b0c75.png)
I've changed the sans-serif.styl file to just including my new font
`$sans-serif = 'CircularStd'
              sans-serif`

For some reason the page still shows the Sans Pro font and I don't even understand where does it take it from.

What do I have to do to change the font?

**Comment by gidhon** [comment]
@Dre-Tas there are possibly two things at play here:

1. `$sans-serif = 'CircularStd', sans-serif` — requires a comma between the font names to work correctly
2. Check if you have perhaps run a production build, it will explain why `Source Sans Pro` is still being shown. Local fonts are only sourced in the development environment by default, the starter template switches to Google's font API for production builds which will still be referencing the old font — unless you have already changed that in your `<head>`
